src/main/java/seedu/address/logic/commands/entry/AddCommand.java

```diff
@@ -46,12 +46,11 @@ public AddCommand(Entry entry) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        if (toAdd.getWord().toString().equals("/stop") || toAdd.getTranslation().toString().equals("/stop")) {
-            throw new CommandException(String.format(MESSAGES_FORBIDDEN, "/stop"));
-        }
-        if (toAdd.getWord().toString().equals("/play") || toAdd.getTranslation().toString().equals("/play")) {
-            throw new CommandException(String.format(MESSAGES_FORBIDDEN, "/play"));
+        if (toAdd.getWord().toString().equals("/stop") || toAdd.getTranslation().toString().equals("/stop")
```

Comment:
Might be better to keep the original if statements. Clearer to read through the code with multiple if statements rather than a long if statement